tests/chart_tests/test_logging_sidecar.py

```diff
@@ -42,3 +44,50 @@ def test_logging_sidecar_config_disabled(self, kube_version):
             show_only="templates/logging-sidecar-configmap.yaml",
         )
         assert len(docs) == 0
+
+    def test_logging_sidecar_custom_config(self, kube_version):
+        """Test logging sidecar config with customConfig flag enabled"""
+        custom_side_carconfig = textwrap.dedent(
```

Comment:
Seems like this var should be either `custom_sidecar_config` or `custom_side_car_config`

Reply:
Thanks. Corrected